appveyor.yml

```diff
@@ -3,33 +3,25 @@ skip_tags: true
 clone_depth: 2
 environment:
   matrix:
-  - JAVA_HOME: C:\Program Files\Java\jdk1.8.0
+    - appveyor_build_worker_image: Visual Studio 2022
+      JAVA_HOME: C:\Program Files\Java\jdk17
 branches:
   only:
-  - master
```

Comment:
this is valid yaml syntax

Reply:
yes. and?

Reply:
unnecessary changes make it harder to read the diff